fdbserver/Status.actor.cpp

```diff
@@ -478,6 +491,8 @@ struct RolesInfo {
 			obj["mutation_bytes"] = StatusCounter(storageMetrics.getValue("MutationBytes")).getStatus();
 			obj["mutations"] = StatusCounter(storageMetrics.getValue("Mutations")).getStatus();
 			obj.setKeyRawNumber("local_rate", storageMetrics.getValue("LocalRate"));
+			obj["fetched_versions"] = StatusCounter(storageMetrics.getValue("FetchedVersions")).getStatus();
+			obj["fetches_from_logs"] = StatusCounter(storageMetrics.getValue("FetchesFromLogs")).getStatus();
```

Comment:
When this is backported, there is an interesting consequence. If you run a cluster that has storage servers at different patch versions (by accident, because you are doing a rolling storage upgrade, etc.), then storage servers at older versions will not have these new fields. When you try to read them here it will throw an error, and the result is that all metrics added to `obj` after these lines will be missing.

To avoid this problem, you can do a couple things. You could put these two lines in a `try/catch` and swallow `error_code_attribute_not_found`, or you could use something like `tryGetValue`.

Reply:
It should be fine to make this change on release-6.3 only and not release-7.0 and master because all patch versions of 7.0+ will log these fields.

Reply:
Makes sense. Thanks for pointing it out.

Done.